openmp/libomptarget/include/omptarget.h

```diff
@@ -312,6 +312,7 @@ int omp_target_memcpy_rect(void *Dst, const void *Src, size_t ElementSize,
                            const size_t *DstDimensions,
                            const size_t *SrcDimensions, int DstDevice,
                            int SrcDevice);
+void *omp_target_memset(void *Ptr, int C, size_t N, int DeviceNum);
```

Comment:
More descriptive argument name for `C` and `N`.

Reply:
I can do that.  I have "borrowed" the argument names from the `memset` interface: `void *memset(void *s, int c, size_t n);`, but I'm OK with being a bit more verbose and use more elaborate names.

Reply:
Resolved via commit 7933bfa0.

Reply:
I'm kinda lost actually. On my side (the "Files changed" tab), I still see `C` and `N`, but you have apparently updated it. Other stuff is similar. I'm not sure what caused this.

Reply:
I guess it's a GitHub PR thing.  The review comments are still attached to the old code that the comment was targeting.  I guess you'd have to go to the "Files Changed" tab of the PR to see the latest changes and get a chance to create a new comment thread.  I'll mark this one here as "resolved" for now.